templates/netlify/remix.init/vscode.json

```diff
@@ -1,9 +1,5 @@
 {
-  "deno.enablePaths": [
-    "globals.d.ts",
```

Comment:
I removed this because it's no longer in our template. See https://github.com/remix-run/remix/pull/3104#discussion_r1032703744